tests/runtime/dwarf

```diff
@@ -45,6 +45,63 @@ REQUIRES_FEATURE dwarf
 TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
 
+# We check the expected output after the timeout.
```

Comment:
What if we instead checked the number of attached probes?
```
EXPECT Attaching X probes...
```

Reply:
Does this work with uprobe_multi? (I haven't got a kernel with that feature to test with)

Reply:
Good point, it doesn't (uprobe_multi will attach just one probe) but we could use `--no-feature=uprobe_multi` to disable it for this test.

Reply:
Moreover, does checking the number of attached probes also checks we're able to intercept their call?

Reply:
Maybe not but there are other tests in this suite to verify that.

Reply:
Reminding this

Reply:
How about this alternative?

Just add this line to the end of the script:
```
interval:ms:10 { if (@timeout) { exit(); } @timeout = 0; if (@count == 1) { @timeout = 1; } }
```

It'll wait 10-20ms after hitting the expected number for `@count` and then exit. That should be plenty of time for any extra function calls to execute if they're going to, and the milliseconds of test overhead will just be lost in the noise.